README.md

```diff
@@ -47,12 +47,13 @@ end
   config.registration.unregister_on_halt = false
 ```
 
-### subscription-manager Configuration
+- **manager** select the registration manager implementation. By default the plugin will use the implementation for the subscription-manager's register command, you can however change that by setting the option to a different implementation:
```

Comment:
I agree, sounds better.

but I would write it as follows:

manager select the registration manager provider. By default the plugin will use the subscription_manager manager <del>provider</del>, you can however change that by setting the option to a different provider:

e.g. in the second sentence leave only manager or provider, otherwise its sounds too complicated